### ISSUE TITLE
Internal Links to Headings or Bookmarks Fail

### ISSUE THREAD

**Issue by jessiezhudev** [wojtekmaj/react-pdf#850]
**Before you start - checklist**

 - [x] I have read documentation in README
 - [x] I have checked [sample](https://github.com/wojtekmaj/react-pdf/tree/master/sample) and [test](https://github.com/wojtekmaj/react-pdf/tree/master/test) suites to see real life basic implementation
 - [x] I have checked if this question is not already asked

**What are you trying to achieve? Please describe.**

I'd like to enable internal links in pdf able to jump to certain Headings or Bookmarks.

**Describe solutions you've tried**

If the internal link jumps to certain pages, `onItemClick` will show right `pageNumber`. However, jump to ceratin Headings or Bookmarks fail.

**Additional information**
<img width="599" alt="Screen Shot 2021-09-13 at 11 39 59 AM" src="https://user-images.githubusercontent.com/18157412/133419285-add9d6dc-16e2-49e6-934b-b2d4e0c36a66.png">

[The pdf I use](https://www.antennahouse.com/hubfs/xsl-fo-sample/pdf/basic-link-1.pdf?hsLang=en)

**Environment**

 - Browser (if applicable) [e.g. Chrome 57, Firefox 59]:
 - React-PDF version [e.g. 3.0.4]: 5.3.2
 - React version [e.g. 16.3.0]: 17.0.2
 - Webpack version (if applicable) [e.g. 4.16.2]:

**Comment by aj47** [comment]
somewhat fixed my issue by using the `onItemClick` property of Document

i.e
```
          <Document
            onItemClick={({pageNumber}) => {
              document.querySelector(`[data-page-number="${pageNumber}"`).scrollIntoView();
            }}
```

**Comment by wojtekmaj** [comment]
Duplicate of #869

Fixed in 5.4.1 and up
